src/include/firebird/IdlFbInterfaces.h

```diff
@@ -6235,17 +6272,22 @@ namespace Firebird
 		}
 
 	public:
-		static const unsigned VERSION = 3;
+		static const unsigned VERSION = 4;
```

Comment:
I wonder why version of ReplicatedSession has been changed while version of ReplicatedRecord wasn't.